script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -1047,17 +1096,20 @@ In that case, there's no way he should still be alive."
 [sha:73f0d414a385e81fe17e7d6453689cd7a0b7310e]{
 -- Page 72, Offset 31877.
 -- 「可能性があるとしたら一つだけだ。俺が一命を取り留めたように、あいつも一命を取り留めた。
-"I can only think of one possible explanation. If I managed to survive by a hair's breadth, then the same must also be true for him.
+"I can only think of one possible explanation.
+ If I was somehow able to escape death, then the same must also have been true for him.
 }
 [sha:13b0a40090673d198855b3a3745e94e4978393d0]{
 -- Page 72, Offset 31878.
 -- ……いや、もしかすると親父は息の根を止めなかったのかもしれないな。
-...No, maybe it's possible that the old man didn't end up killing him after all.
+...On second thought, it's also possible that the old man just didn't actually kill him at all.
 }
 [sha:d2faa427ada65fb457c50431f6b75d3deed0efb9]{
 -- Page 72, Offset 31879.
 -- どんなに狂ってしまっても血を分けた自分の息子だ。だから殺せず、人目につかないところで養生させていたんじゃないのか？　俺が病院に担ぎこまれたみたいに」
-No matter how mad SHIKI has become, he's still his son by blood. Then wouldn't he let him live and take care of him in secret? Like how I was taken to the hospital."
+Even if SHIKI had gone mad, he was still his own flesh and blood.
+ What if instead of killing him, he spirited him away somewhere to try and cure him in secret?
```

Comment:
I'm hesitant to introduce a japanese-derived phrase in a TL of a sentence that does not use it. Maybe "whisked him off somewhere"